### DISCUSSION TITLE
how to reuse hold page with another route path?

### DISCUSSION THREAD

**Discussion by Genuifx** [posva/unplugin-vue-router#551] [Questions and Help] [answered]
how to reuse the hold page with another route path?

for example: A page in `/page/to/a`, I need the `/page/to/b` path to use A page code. Rewrite is not the choice, cause the A page need keepalive itself, also I want the ```path/to/b``` keepalive

**Comment by posva** [comment, marked as answer]
Define an alias `definePage({ alias: '/page/to/b' })` if it's meant to be the same page. If not, define two pages and abstract the common parts in a component that is displayed in both pages.
